src/sentry/eventstream/kafka.py

```diff
@@ -35,12 +40,7 @@ def delivery_callback(self, error, message):
         if error is not None:
             logger.warning('Could not publish event (error: %s): %r', error, message)
 
-    def publish(self, group, event, is_new, is_sample, is_regression, is_new_group_environment, primary_hash, skip_consume=False):
-        project = event.project
-        retention_days = quotas.get_event_retention(
-            organization=Organization(project.organization_id)
-        )
-
+    def _send(self, project_id, _type, *data):
```

Comment:
More than fair, it was just weird to handle since `insert` is the only one sending multiple "`data`" objects down. I've changed it some, lmk if I'm missing something that's more clear.